analytics.js

```diff
@@ -1278,7 +1291,11 @@ parseQueryString(window.location.search);
 
 if (isValidArgsArray) argumentsArray.forEach((x) => instance.toBeProcessedArray.push(x));
 
-processDataInAnalyticsArray(instance);
+if (instance.toBeProcessedArray.length && instance.toBeProcessedArray[0][0] === defaultMethod) {
```

Comment:
Extract the load event in the while block above and execute here directly.

Reply:
Addressed